controllers/metal3machinetemplate_controller_test.go

```diff
@@ -70,7 +73,98 @@ var _ = Describe("Metal3MachineTemplate controller", func() {
 			Namespace: namespaceName,
 		},
 	}
+	type TestCaseM3MoM3MT struct {
+		M3Machine     *capm3.Metal3Machine
+		M3MTemplate   *capm3.Metal3MachineTemplate
+		ExpectRequest bool
+	}
+	DescribeTable("Cluster To Metal3Machines tests",
+		func(tc TestCaseM3MoM3MT) {
+			r := Metal3MachineTemplateReconciler{}
+			obj := client.Object(tc.M3Machine)
+			reqs := r.Metal3MachinesToMetal3MachineTemplate(obj)
 
+			if tc.ExpectRequest {
+				Expect(len(reqs)).To(Equal(1), "Expected 1 request, found %d", len(reqs))
+				Expect(tc.M3Machine.Annotations[clonedFromName]).To(Equal(tc.M3MTemplate.Name))
+				Expect(tc.M3Machine.Annotations[clonedFromGroupKind]).To(Equal("Metal3MachineTemplate.infrastructure.cluster.x-k8s.io"))
```

Comment:
Should we have a const = "Metal3MachineTemplate.infrastructure.cluster.x-k8s.io" and use it instead anywhere needed?

Reply:
This has been used in m3mt controller code as well, I would suggest making it const there and reuse it here